### PR TITLE
Basic Grid Docs

### PR DESCRIPTION
**Pull request by koehlerson**
I started to document `Grid` manual with a lot of copy pasta :spaghetti: of this issue https://github.com/KristofferC/JuAFEM.jl/issues/277

Is there a way to preview the deployed docs? I tested it locally, but I'm always suspicious, if graphic paths work as they should

### CODE REVIEW

**Comment by fredrikekre** [comment]
Great, thanks!

> Is there a way to preview the deployed docs?

Only if you create the PR from a branch on this repo instead of your own fork. I think Kristoffer gave you access?

**Comment by KristofferC** [comment]
> I think Kristoffer gave you access?

Yes, I (hopefully) invited you as a collaborator so you should have push access to the repo once accepting it.

**Comment by koehlerson** [comment]
Ok good to know. Yes, I received the invitation last week, thanks for this! 

So, suppose I have the PR from a branch of KristofferC/JuAFEM.jl, can I check the doc CI or where can I find the deployed docs of the PR? Sorry, probably a obvious answer, but I don't know better since I'm lacking collaboration experience. Regarding merge doc changes, should I just merge, when I think it's OK or should I wait for a review as for normal src code changes?

**Comment by fredrikekre** [comment]
> So, suppose I have the PR from a branch of KristofferC/JuAFEM.jl, can I check the doc CI or where can I find the deployed docs of the PR?

It will be posted as a CI status with a link, see e.g. the `documenter/deploy` status on this PR: https://github.com/KristofferC/JuAFEM.jl/pull/301

File: docs/src/manual/grid.md [hunk 1]
```diff
@@ -3,8 +3,86 @@ DocTestSetup = :(using JuAFEM)
...
+
+The highlighted face, i.e. the two lines from node ID 3 to 6 and from 6 to 9, on the right hand side of our test mesh can now be described as
+
+```julia
+julia> edges = [
+           (3,6),
+           (6,9)
+       ]
+
+julia> interpolation = Lagrange{2, RefTetrahedron, 1}()
```
Comment (fredrikekre) [inline docs/src/manual/grid.md:59]: This looks a bit out of place. I suggest you move this to after the `compute_faceset` function in the next code block.

**Comment by fredrikekre** [comment]
How did you generate the svg's?

**Comment by koehlerson** [comment]
The svgs are created with Inkscape. If you don't like something w.r.t. the svgs I can change them as well. 

I thought it's a good idea to use something that is FOSS and runs without any big problems on Linux, OSX and Windows (I was too lazy to use tikz)

File: docs/src/manual/grid.md (hunk 1 above)
Comment (koehlerson) [reply docs/src/manual/grid.md]: yep, agree

**Comment by fredrikekre** [comment]
Yea I was just curious. It looks good to me, and certainly better than the non-existent docs.

**Pull request merged**